### ISSUE TITLE
yolov8 does not put color on images for segment application

### ISSUE THREAD

**Issue by hanshantong** [ultralytics/ultralytics#1017]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, everyone, I train my own dataset using yolov8 (8.0.39) for segmentation. The trained image results have boxes but don't put any color at different region for distinguishing. I have checked the document from official website and the "default.yaml' file, but don't find any parameter related to that. In yolov5, it has parameters to toggle boxes showing and color showing in images. In yolov8, how should I do that. Is anyone encounter such issue or requirement?

My environments:
os: ubuntu 18.04
python: 3.8
yolov8 version: 8.0.39
dataset: self-defined.
related snippet code:
```from ultralytics import YOLO

model = YOLO(model="./yolov8n.pt", type='v8')
model.task = "segment"
model.train(data="./mydataset.yaml",
            epochs=3,
            device="0, 2, 3",
            pretrained=True,
            batch=18,
            save=True,
            save_txt=True,
            # save_dir="./runs/segment/train",
            line_thickness=100,
            verbose=True)
```

**Comment by glenn-jocher** [comment]
Here are some suggestions to improve the quality of the generated segmentation mask:

Increase the training data: Having more training data, especially for the specific objects you are trying to detect and segment, can help improve the quality of the segmentation mask.

Use more advanced augmentation techniques: Augmentations like random crops, random rotations, and color jitter can help the model learn more robust features and improve the accuracy of the segmentation mask.

Adjust hyperparameters: Try adjusting the learning rate, batch size, and other hyperparameters to optimize the model's performance.

Experiment with different architectures: Some architectures might be better suited for your specific problem than others. Consider trying different architectures and see which one gives you the best results.

Fine-tune the model: If you have a pre-trained model, fine-tuning it on your specific dataset can improve the quality of the segmentation mask.

Use post-processing techniques: Post-processing techniques like morphological operations and contour detection can help refine the segmentation mask and extract the object contour more accurately.

Use a combination of detection and segmentation: In some cases, using a combination of object detection and segmentation can help improve the accuracy of the segmentation mask. For example, you can use object detection to identify the objects in the image and then use segmentation to extract the exact contour of the objects.
